usage/ui.md

```diff
@@ -110,6 +110,98 @@ import Modal from "@ui/js/modal";
+
+| Props   | Default | Description                                                            |
+|---------|---------|------------------------------------------------------------------------|
+| success | `false` | If set it will add a green background to the row                       |
+| info    | `false` | If set it will add a background according to the main color to the row |
+| danger  | `false` | If set it will add a red background to the row                         |
+| warning | `false` | If set it will add a yeallo background to the row                      |
+| tooltip | `false` | If set it will add a tippy tooltip                                     |
+
+#### Table `<x-art-tables.header` props
+
+| Props      | Default | Description                                                                                       |
+|------------|---------|---------------------------------------------------------------------------------------------------|
+| responsive | `false` | If set it will hide the column according on the breakpoint that is added on the `breakpoint` prop |
+| breakpoint | `lg`    | In which breakpoint it will hide the column                                                       |
+| firstOn    | `null`  | In which screen sizes this column will be the first one (`xl`, `lg`, etc)                         |
+| lastOn     | `null`  | In which screen sizes this column will be the last one (`xl`, `lg`, etc)                          |
+| class      | `''`    | Column CSS class                                                                                  |
+| name       | `''`    | If set it will use the laravel `@lang`  helper to get the value of the column                     |
+
+#### Table `<x-art-tables.cell` props
+
+| Props      | Default | Description                                                                                       |
+|------------|---------|---------------------------------------------------------------------------------------------------|
+| responsive | `false` | If set it will hide the column according on the breakpoint that is added on the `breakpoint` prop |
+| breakpoint | `lg`    | In which breakpoint it will hide the column                                                       |
+| firstOn    | `null`  | In which screen sizes this column will be the first one (`xl`, `lg`, etc)                         |
+| lastOn     | `null`  | In which screen sizes this column will be the last one (`xl`, `lg`, etc)                          |
+| class      | `''`    | Column CSS class                                                                                  |
+| colspan    | `null`  | `td` colspan attribute                                                                            |
+
+
+```bash
+yarn add body-scroll-lock
+yarn add focus-trap
+```
+
+1. Import the modal script in your `resources/js/app.js` file
+
+```js
+import Modal from "@ui/js/modal";
+
+window.Modal = Modal;
+```
+
```

Comment:
I think this part is related to the previous `### Modals` section 😅

Reply:
😅